docs/UsersGuide/source/BuildRunSRW.rst

```diff
@@ -0,0 +1,847 @@
+
+.. _PredefinedGrids:
+
+.. table::  Predefined grids in the SRW App
+
+   +----------------------+-------------------+--------------------------------+
+   | **Grid Name**        | **Grid Type**     | **Quilting (write component)** |
+   +======================+===================+================================+
+   | RRFS_CONUS_25km      | ESG grid          | lambert_conformal              |
+   +----------------------+-------------------+--------------------------------+
+   | RRFS_CONUS_13km      | ESG grid          | lambert_conformal              |
+   +----------------------+-------------------+--------------------------------+
+   | RRFS_CONUS_3km       | ESG grid          | lambert_conformal              |
+   +----------------------+-------------------+--------------------------------+
+
+
+.. _GenerateForecast:
+
+Generate the Forecast Experiment 
+=================================
+Generating the forecast experiment requires three steps:
+
+* :ref:`Set experiment parameters <ExptConfig>`
+* :ref:`Set Python and other environment parameters <SetUpPythonEnv>`
+* :ref:`Run a script to generate the experiment workflow <GenerateWorkflow>`
+
+The first two steps depend on the platform being used and are described here for each Level 1 platform. Users will need to adjust the instructions to their machine if they are working on a Level 2-4 platform. Information in :numref:`Chapter %s: Configuring the Workflow <ConfigWorkflow>` can help with this. 
+
+.. _ExptConfig:
+
+Set Experiment Parameters
+---------------------------- 
+
+Each experiment requires certain basic information to run (e.g., date, grid, physics suite). This information is specified in ``config_defaults.sh`` and in the user-specific ``config.sh`` file. When generating a new experiment, the SRW App first reads and assigns default values from the ``config_defaults.sh`` file. Then, it reads and (re)assigns variables from the user's custom ``config.sh`` file. For background info on ``config_defaults.sh``, read :numref:`Section %s <DefaultConfigSection>`, or jump to :numref:`Section %s <UserSpecificConfig>` to continue configuring the experiment. 
+
+.. _DefaultConfigSection:
+
+Default configuration: ``config_defaults.sh``
+------------------------------------------------
+
+.. note::
+   This section provides background information on how the SRW App uses the ``config_defaults.sh`` file. This information is helpful but not essential to running the SRW App. Users may skip to :numref:`Step %s <UserSpecificConfig>` to continue configuring their experiment. 
```

Comment:
Consider changing to something like "This information is helpful and informative but not essential to modify prior to running the SRW App".